docs/resources/azurerm_ad_users.md

```diff
@@ -0,0 +1,100 @@
+---
+title: About the azure_resource_groups Resource
+platform: azure
+---
+
+# azurerm\_ad\_users
+
+Use the `azurerm_ad_users` InSpec audit resource to test properties of
+some or all Azure Active Directory users within a Tenant.
+<br />
+
+## Syntax
+
+An `azurerm_ad_users` resource block returns all Azure Active Directory user accounts for the 
+contained within the configured Tenant and then tests that group group of users.
+
+    describe azurerm_ad_users do
+      ...
+    end
+<br />
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+### Check Users are present
+
+    describe azurerm_ad_users do
+      it { should exist }
+    end
+
+### Ensure there are no Guest accounts active 
+
+    describe azurerm_ad_users.guest_accounts
+      it { should_not exist }
+    end
+<br />
+
+## Filter Criteria
+
+* `names`
+
+### names
+
+Filters the results to include only those resource groups that match the given
+name. This is a string value.
+
+     describe azurerm_ad_users.where{ displayName.eql?('Joe Bloggs') } do
+      it { should exist }
+    end
+
+## Properties
+
+* `azureIds`
+* `displayNames`
+* `mails`
+* `userTypes`
+
+### azureIds
+
+The azureIds property provides a list of all User's Azure IDs.
+
+    its('azure_ids') { should include '44211066-f292-4546-8ced-2ab0e0911f44' }
+    
+### displayNames
+
+The displayNames property provides a list of all the User display names.
```

Comment:
Should we match this with to say `display_names` like the example or should it be `displayNames`?

Reply:
The column in the FilterTable is indeed `display_names` - will update example!